src/views/StatusBoard.vue

```diff
@@ -156,7 +182,10 @@ export default {
     isCreatingNotification: false,
     isExporting: false,
     isCurrentTermLocked: false,
-    selectedDepartmentIds: []
+    selectedDepartmentIds: [],
+    currentTermId: null,
+    currentTermName: null,
+    availableTerms: []
```

Comment:
@cesarvh - minor issue: We tend to alphabetize variable declarations and imports.